modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -318,6 +354,11 @@ class Tolerance_FloatRel_IntAbs : public Wrappable<Tolerance_FloatRel_IntAbs>
             }
         }
     }
+    std::string desc() {
```

Comment:
y desc?

I believe this method could take an `ostream&` as parameter and write its stuff there.

Then there's no matter if it is a std::cout our some stringstream - we use that stream API native to write our stuff out.

Reply:
makes sense, will update

Reply:
I suggest to overload `operator<<` then. It seems to be the most robust option for such stuff + we get printability of comparators even without introduced `CompareF`

Reply:
you can overload that `operator<<` using that `ostream&`-oriented method. Bingo!

Reply:
Going further:

```c++
template<typename T> PP { const T &t; };
template<typename T> std::ostream& operator<< (std::ostream&os, const PP& pp) {
    pp.print_desc(os);
    return os;
}
```
So then you can dump a whatever tolerance with 
```c++
... << PP{tol} << ...
```
Probably it works even this way.

Reply:
But you already did the operators - ok!